azkaban_cli/azkaban_cli.py

```diff
@@ -438,6 +439,16 @@ def __fetch_executions_of_a_flow(ctx, project, flow, start, length):
     except FetchExecutionsOfAFlowError as e:
         logging.error(str(e))
 
+@login_required
+def __resume_flow_execution(ctx, execution_id):
+    azkaban = ctx.obj[u'azkaban']
+
+    try:
+        json = azkaban.resume_flow_execution(execution_id)
+        __log_flow_execution_updates(json)
```

Comment:
The response is empty (`{}`) or an error. You don't need to log anything, maybe a success message (but then we would need to add one for the other functions that don't return anything either)

Reply:
done

Reply:
See my new review